python-main.js

```diff
@@ -1392,6 +1390,13 @@ function web_editor(config) {
         REPL.prefs_.set('font-size', getFontSize());
     }
 
+    function openModalMsg(title,content,helplink){
```

Comment:
Can add a third parameter here to be an object to define the action buttons?
The function call could be something like:

```js
openModalMsg(
    'My Title',
    'My Content',
    {
        'some link': 'http://www.google.com',
        'second link': 'http://microbit.org',
        'close': 'close'
    }
);
```

Where a "close" value would close the modal (so we would need to check for that string and add the appropriate js call to hide the modal).